README_WINDOWS.md

```diff
@@ -44,4 +44,14 @@ PowerShell's execution policy prevents you from running the client's installatio
 
     Set-ExecutionPolicy -ExecutionPolicy Unrestricted -Scope Process
 
-For further information, please refer to the PowerShell documentation [About Execution Policies](https://docs.microsoft.com/en-sg/powershell/module/microsoft.powershell.core/about/about_execution_policies).
\ No newline at end of file
+For further information, please refer to the PowerShell documentation [About Execution Policies](https://docs.microsoft.com/en-sg/powershell/module/microsoft.powershell.core/about/about_execution_policies).
+
+#### Distutils has been removed from Python 3.12, which causes my node-gyp build to fail. How can I fix this?
+
+Since Distutils is no longer include in Python after 3.12, it is necessary to install them yourself if using Python 3.12 or above.
```

Comment:
```suggestion
Since Distutils is no longer included in Python after 3.12, it is necessary to install Distutils yourself if using Python 3.12 or above.
```